shell/client/admin/user-details.html

```diff
@@ -327,11 +327,16 @@
           <strong>Account deletion is permanent and irreversible.</strong>  Please be certain.
```

Comment:
Probably the same copy adjustments suggested above make sense here, with changes as appropriate to refer to `{{guessUserName}}` rather than "you" and "your grains"